editoast/src/views/timetable/stdcm.rs

```diff
@@ -251,7 +255,38 @@ async fn stdcm(
 
     let stdcm_response = stdcm_request.fetch(core_client.as_ref()).await?;
 
-    // 6. Handle STDCM Core Response
+    // 6. Check if the current tracing level is debug or lower, and if so, log STDCM request and response
+    if tracing::level_filters::LevelFilter::current() >= tracing::Level::DEBUG {
+        let user_id = auth.authorizer()?.user_id();
+        let stdcm_response_for_spawn = stdcm_response.clone();
```

Comment:
You can inline this (no need for a new variable)... unless everything is moved inside the closure.